README.md

```diff
@@ -94,6 +107,31 @@ This plugin also supports flight recording (see next section)
 --tf-replay=[record|replay|disable]
 ```
 
+### Teardown Options
+
+`pytest_terraform` supports three different teardown modes for the terraform decorator.
+The default, `pytest_terraform.options.teardown.ON` will always attempt to teardown any and all modules via `terraform destory`.
+If for any reason destroy fails it will raise an exception to alert the test runner.
+The next mode, `pytest_terraform.options.teardown.SILENT`, will invoke `terraform destroy` as with `teardown.ON` but will silence any failures.
+This mode is particularly help if your test function performs destructive actions against any objects created by the terraform module.
+The final option is `pytest_terraform.options.teardown.OFF` which will remove the teardown method register all together.
+This should generally only be used in very specific situations and is considered an edge case.
+
+There is a special `pytest_terraform.options.teardown.DEFAULT` which is what the `teardown` parameter actually defaults to.
+This "default" option can be overriden either at runtime via the CLI or pytest ini with:
```

Comment:
whats a scenario where a user executing the tests would do this? filtered to a single test i could see utility, but there's a knowledge imbalance to someone executing tests vs the test author, and this seems like an issue with the test code itself, and likely to leave garbage in the env if used incorrectly. just trying to understand the usage scenarios on this flag.

Reply:
This may be a bit over-engineered then. I'm happy to drop a lot of the option resolver bits. The use case I had in my mind was as a test author I wanted to inject a command line argument to leave behind the artifacts to inspect, IE:

```
pytest tests -k tests_item --tf-teardown=off
```

The same can be achieved by just editing the test wrapper though. This also won't change anything if the author has already modified it from default. The ini was put in place in case this feature ever grows and there's a better "default" option added authors could wholesale configure to a different style.

Again, I think a bit over engineered for this use case happy to drop for now